packages/docs/src/docs/try/components/hegel.js

```diff
@@ -100,6 +107,108 @@ export async function getDiagnostics(sourceCode) {
   return errors.map(toTransferableObject);
 }
 
+/**
+ * Summon and format completion items.
+ * @param {import("monaco-editor").languages.CompletionItemKind} completionItemKind 
```

Comment:
Is this supported by `hegel` itself :heart_eyes: or is this typescript syntax ?

Reply:
At current moment Hegel does not support comments. It is just for TypeScript IntelliSense.